generator/src/main/java/com/scottlogic/deg/generator/guice/VisualiserFactoryProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.scottlogic.deg.generator.guice;
+
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.scottlogic.deg.generator.generation.GenerationConfigSource;
+import com.scottlogic.deg.generator.generation.visualiser.VisualiserFactory;
+import com.scottlogic.deg.generator.generation.visualiser.VisualiserWriterFactory;
+
+public class VisualiserFactoryProvider implements Provider<VisualiserFactory> {
```

Comment:
visualiser factory seems like it could be created without the need for a provider

Reply:
yes it can.  will get rid of